app/assets/stylesheets/layout_old/home/main.scss

```diff
@@ -14,14 +14,20 @@
     overflow: visible;
     z-index: 100;
   }
+  &.homepage #mastheadNavigation { // Importants added because of display: block inline style -SH
+    display: none !important;
+    @include bp(wildcard-min, 680px) {
+      display: none !important;
```

Comment:
Is it really necessary to have this in here twice (line 18 and 20)? If you just say `display: none !important` outside the media query it should apply to all breakpoints by default.

Reply:
We are doing this because there is an inline display block on the masthead navigation (#mastheadNavigation) and we have to override it to hide it. We believe this style attribute is being added with JavaScript regarding a plugin for the masthead slideshow.